sal_interfaces/MTHexapod/MTHexapod_Telemetry.xml

```diff
@@ -18,6 +18,13 @@
       <Units>unitless</Units>
       <Count>6</Count>
     </item>
+    <item>
+      <EFDB_Name>positionError</EFDB_Name>
+      <Description>Position error of each MTHexapod actuator (x1,x2,x3,x4,x5,x6) in counts</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>unitless</Units>
```

Comment:
I think the unit is `micron` instead of `counts`:
1. https://github.com/lsst-ts/ts_hexapod_controller/blob/develop/include/interface/telemetryStructure.h#L132-L141
2. https://github.com/lsst-ts/ts_mthexapod/blob/develop/python/lsst/ts/mthexapod/structs.py#L114-L116

Reply:
Great! Thanks!